framework/src/main/java/org/apache/drill/test/framework/Utils.java

```diff
@@ -169,7 +170,21 @@ public static ConnectionPool getConnectionPool() {
         boolean foundTests = false;
         for (String testGroup : testGroups) {
           if (categories != null && !categories.contains(testGroup)) {
-            continue;
+            if (categories.contains("smoke") &&
```

Comment:
Perhaps we should address this too, then.  I went through almost all of the json files, and I saw mostly smoke.  I saw a few negative and other ones, but very few.  I would be happy to go through them all again and make them more consistent if possible.